aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1966,8 +1966,12 @@ void scaled_gemm(
             }
   #endif
   }
-#else
-  // rowwise isn't supported using cublaslt or older hipblaslt
+#elif (CUDA_VERSION < 12080) || defined(USE_ROCM)
```

Comment:
As per @jagadish-amd's suggestion.
```suggestion
#elif (CUDA_VERSION < 12080) && !defined(USE_ROCM)
```